src/Autowhatever.js

```diff
@@ -30,6 +30,10 @@ export default class Autowhatever extends Component {
     shouldRenderSection: PropTypes.func,   // This function gets a section and returns whether it should be rendered, or not.
     renderSectionTitle: PropTypes.func,    // This function gets a section and renders its title.
     getSectionItems: PropTypes.func,       // This function gets a section and returns its items, which will be passed into `renderItem` for rendering.
+    inputElement: PropTypes.oneOfType([    // This element is used to render the input
+      PropTypes.func,
+      PropTypes.string
+    ]),
```

Comment:
Unfortunately `PropTypes.node` doesn't include functional stateless components, but `PropTypes.node` + `PropTypes.func` should cover it (that will cover string as well).

```js
// Anything that can be rendered: numbers, strings, elements or an array
// (or fragment) containing these types.
optionalNode: React.PropTypes.node,
```

Or is that too lenient? Would `func` + `element` work or doesn't that work for `'input'`?

Thanks for taking over @angelo-v, i was just gonna make the changes, but i guess you got it.